docs/docs/hydroflow/quickstart/example_4_neighbors.md

```diff
@@ -140,13 +139,13 @@ don't have any corresponding value, so we feed `origin` through a [`map()`](../s
 to generate `(v, ())` elements as the first join input. 
 
 The `stream_of_edges` are `(src, dst)` pairs,
-so the join's output is `(src, ((), dst))` where `dst` are new neighbor
+so the join's output is `(src, ((), dst))` where `dst` values are new neighbor
 vertices. So the `my_join` variable feeds the output of the join through a `flat_map` to extract the pairs into 2-item arrays, which are flattened to give us a list of all vertices reached.
 Finally we print the neighbor vertices as follows:
 ```rust,ignore
     my_join -> unique() -> for_each(|n| println!("Reached: {}", n));
 ```
-The [unique](../syntax/surface_ops.gen.md#unique) operator removes duplicates from the stream to make things more readable. Note that `unique` does not run in a streaming fashion, which we will talk about more [below](#strata-and-ticks).
+The [unique](../syntax/surface_ops.gen.md#unique) operator removes duplicates from the stream to make things more readable. Note that `unique` runs in a streaming fashion, which we will talk about more [below](#strata-and-ticks).
```

Comment:
"Which we will talke about more" doesn't make sense here anymore.
And the link is broken anyway, would need to be `./example_6_unreachability#strata-and-ticks`